libs/cart/package.json

```diff
@@ -34,8 +34,9 @@
     "angular-in-memory-web-api": "0.0.0-PLACEHOLDER",
     "apollo-angular": "0.0.0-PLACEHOLDER",
     "apollo-client": "0.0.0-PLACEHOLDER",
-    "graphql-tag": "0.0.0-PLACEHOLDER"
-  },
+    "graphql-tag": "0.0.0-PLACEHOLDER",
+		"@daffodil/driver": "0.0.0-PLACEHOLDER"
+	},
   "devDependencies": {
```

Comment:
For lerna, `@daffodil/driver` needs to be in the `devDependencies` as well.

Reply:
done